lldb/test/API/functionalities/fork/concurrent_vfork/main.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <mutex>
+#include <sys/wait.h>
+#include <thread>
+#include <unistd.h>
+#include <vector>
+
+pid_t g_pid = 0;
+std::mutex g_child_pids_mutex;
+std::vector<pid_t> g_child_pids;
+
+int call_vfork(int index) {
+  pid_t child_pid = vfork();
```

Comment:
Use either `fork()` of `vfork()` depending on the value of the `use_vfork` argument:
```
pid_t child_pid = use_vfork ? vfork() : fork();
```